src/words/ast/INodeCreateClass.java

```diff
@@ -10,7 +15,40 @@ public INodeCreateClass(Object... children) {
 
 	@Override
 	public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException {
-		// TODO
-		throw new AssertionError("Not yet implemented");
+		ASTValue className  = this.children.get(0).eval(environment);
+		ASTValue parentClassName = this.children.get(1).eval(environment);
+		
+		WordsClass wordsClass = environment.createClass(className.stringValue, parentClassName.stringValue);
+		
+		INodeClassStatementList statementList = (INodeClassStatementList) this.children.get(2);
+		if (statementList != null) {
+			for (Object child : statementList.children) {
```

Comment:
This part would be an opportunity to leverage the inherited attributes component of the queue-prop-assign branch in #87, so that the implementation of INodeClassStatementList is separated out (and it would receive the class object is an inherited attribute).